spec/models/workflow_step_spec.rb

```diff
@@ -22,6 +22,22 @@
       expect(subject.valid?).to be false
     end
   end
+  context 'without valid status' do
+    it 'is not valid if the status is nil' do
+      subject.status = nil
+      expect(subject.valid?).to be false
+    end
+    it 'is not valid if the status is a bogus value' do
+      subject.status = 'bogus'
+      expect(subject.valid?).to be false
+    end
+  end
+  context 'status is case insensitive' do
+    it 'is valid if the status is valid but all caps' do
+      subject.status = 'COMPLETED'
```

Comment:
I don't understand why we'd want this to be valid.

Reply:
BECAUSE SOMETIMES PEOPLE LIKE TO SHOUT?

but seriously, i don't think its a requirement

Reply:
:hear_no_evil: :laughing: